src/plugins/vis_type_markdown/public/markdown_vis.ts

```diff
@@ -23,10 +23,10 @@ export const markdownVisDefinition: VisTypeDefinition<MarkdownVisParams> = {
   icon: 'visText',
   group: VisGroups.TOOLS,
   titleInWizard: i18n.translate('visTypeMarkdown.markdownTitleInWizard', {
-    defaultMessage: 'Text',
+    defaultMessage: 'Markdown text',
   }),
   description: i18n.translate('visTypeMarkdown.markdownDescription', {
-    defaultMessage: 'Add text and images to your dashboard.',
+    defaultMessage: 'Legacy tool for adding text or images to dashboards.',
```

Comment:
We aren't currently considering the markdown as "legacy" on the Dashboard side since we don't have a replacement - can you explain why this was included? :) 

![Screenshot 2024-10-15 at 8 58 47 AM](https://github.com/user-attachments/assets/cb38810f-6149-4a7c-8da3-8a34a00c0d77)

Reply:
I moved Markdown Text for deprecation just to have the code prepared once we have the decision. I'm with Hannah on it, anything is fine, but let's keep it consistent.

Reply:
Hey, Markdown is one of the most used panels in dashboards, it will be weird for users to see this is in Legacy without any other replacement and it will be hard for them to find. I think **we should keep it where it is now in the dashboard menu**. 

I thought it was fine to move it to the Legacy bucket in the Visualizations menu because I assume almost no one is creating Markdowns from there (since they don't make sense without dashboards) and that way it's ready when we have the inline markdown in dashboards. But you are right it's inconsistent so my ideal solution will be **totally removing the Markdown creation option from the Visualize Library**. Will that work?

Reply:
This is how the legacy tab would look like if we do that:
<img width="731" alt="Screenshot 2024-10-18 at 14 23 23" src="https://github.com/user-attachments/assets/c0a61c7f-f2c0-444e-af3d-ff9cbb362c40">

Reply:
Thanks Marta, I think this is the best solution. @Heenawter @MichaelMarcialis any concerns?

Reply:
@teresaalvarezsoler @mbondyra This works for me! :)